hydra/_internal/utils.py

```diff
@@ -204,7 +205,14 @@ def run_and_report(func: Any) -> Any:
         if "HYDRA_FULL_ERROR" in os.environ and os.environ["HYDRA_FULL_ERROR"] == "1":
             print_exc()
         else:
-            if isinstance(ex, (ConfigCompositionException, OverrideParseException)):
+            if isinstance(
+                ex,
+                (
+                    ConfigCompositionException,
+                    OverrideParseException,
+                    InstantiateErrorException,
```

Comment:
2 is a party, 3 is too much.
create a common base class for those.
CompactHydraException ?

Reply:
Fixed in https://github.com/facebookresearch/hydra/pull/820/commits/6e683536666640a4470f8520946c2860b77219aa